components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -234,6 +234,12 @@ be lost if SCREAM_HACK_XML is not enabled.
       <Ckm type="real" doc="Eddy diffusivity coefficient for momentum">0.1</Ckm>
     </shoc>
 
+    <!-- MAM4xx-ACI -->
+    <mam4_aci inherit="atm_proc_base">
+      <wsubmin type="real" doc="Minimum diagnostic sub-grid vertical velocity">0.001</wsubmin>
```

Comment:
That is a good point. I will fix it in the C++ code.

Reply:
The (characteristic) updraft velocity (wsub) is dynamically calculated in the model. wsubmin is used to apply a lower bound for wsub. In E3SMv2 aerosol activation code (ndrop), there is an additional lower bound 0.1 m/s for wsub. It would be nice to have a single namelist switch to control this lower bound, which is often used for model tuning and testing.

Reply:
So `wsub>=max(wsubmin,0.1)`? Then it makes no sense to set `wsubmin` to `0.001` in the defaults, no? I would just set `0.1`.

Reply:
Yes. The changes needed are 1) remove the wsub lower bound (0.1) in ndrop; and 2) set wsubmin = 0.1 in the default namelist.